punjab/httpb.py

```diff
@@ -6,7 +6,8 @@
 from twisted.internet import defer, task
 from twisted.python import log
 
-from zope.interface import Interface, implements
+
+from zope.interface import Interface, implementer
```

Comment:
Does this change the requirement for zope.interface? I just realized there is no requirements.txt for punjab. :)

Reply:
According to https://github.com/zopefoundation/zope.interface/blob/master/CHANGES.rst you need at least 3.6.0 for it, which is roughly nine years old.
I will add a requirements.txt